### PR TITLE
Updating regularly_used_data.rst

### LINKED ISSUE #2006: Text and structure change for better visibility
Reading through the schema tables was a bit hard with the way the titles were displayed. A change to title headings would make reading through the [list of data tables](https://oss-augur.readthedocs.io/en/main/schema/regularly_used_data.html) much easier to see.

The typeface of some commands do not match the rest. This is a minor change in the [Make Commands](https://oss-augur.readthedocs.io/en/main/development-guide/make/development.html) page.

Warning and note signs could be different symbols. Both have an exclamation mark and the title after, but at first glance, they seem the same. Maybe have an asterisk for a note and keep the exclamation for warnings.
image

For context:
unnamed

unnamed (2)

### PR DESCRIPTION
**Pull request by witchtrish**
Signed-off-by: Trish Nguyen <n.trish00@gmail.com>

- Updating the rst markdown to bolden each command for higher visibility.

This PR fixes #2006 

- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by sgoggins — DISMISSED** [review]
LGTM

File: docs/source/schema/regularly_used_data.rst [hunk 1]
```diff
@@ -3,14 +3,14 @@ List of Regularly Used Data Tables In Augur
 
 **This is a list of data tables in augur that are regularly used and the various workers attached to them.**
 
-    * Commits - This is where a record for every file in every commit in every repository in an Augur instance is kept. 
+    **Commits** - This is where a record for every file in every commit in every repository in an Augur instance is kept. 
         
-        * Worker: Facade worker collects, and also stores platform user information in the commits table. 
+        * * Worker * : Facade worker collects, and also stores platform user information in the commits table.
```
Comment (ABrain7710) [inline docs/source/schema/regularly_used_data.rst:8]: I believe there is a missing * here on line 8

**Review by Preshh0 — COMMENTED** [review]
@witchtrish," * Contributor_breadth_worker * " on line 24, is it supposed to be bold? If yes, the markup isn't structured properly to make it bold. Could you fix that? 
Otherwise, it looks good to me.

File: docs/source/schema/regularly_used_data.rst [hunk 2]
```diff
@@ -19,15 +19,15 @@ List of Regularly Used Data Tables In Augur
                 .. image:: images/contributor_affiliations.png
                    :width: 200
 
-    * Contributor_repo - Storage of a snowball sample of all the repositories anyone in your schema has accessed on GitHub. So, for example, if you wanted to know all the repositories that people on your project contributed to, this would be the table. 
+    **Contributor_repo** - Storage of a snowball sample of all the repositories anyone in your schema has accessed on GitHub. So, for example, if you wanted to know all the repositories that people on your project contributed to, this would be the table. 
         
-        * Contributor_breadth_worker populates this table
+        * * Contributor_breadth_worker * populates this table
```
Comment (Preshh0) [inline docs/source/schema/regularly_used_data.rst:24]: Here.

Comment (Preshh0) [inline docs/source/schema/regularly_used_data.rst:24]: Here.

**Review by Preshh0 — APPROVED** [review]
LGTM👌

**Review by sgoggins — APPROVED** [review]

**Pull request merged**